CHANGELOG.md

```diff
@@ -2,6 +2,245 @@
 - [v1.0.0 - v1.9.10](CHANGELOG-pre-v1.10.md)
 - [v0.11.6 and earlier](CHANGELOG-v0.md)
 
+## 1.17.0-RC1
+### May 30, 2024
+CHANGES:
```

Comment:
```suggestion

CHANGES:

```
I think you'll need a space here for this to render correctly, and for the rest of the headings in the RC1 entry.